### DISCUSSION TITLE
Help! This loading step took too much time.

### DISCUSSION THREAD

**Discussion by Decccc** [SYSTRAN/faster-whisper#978] [Q&A]
This step took too much time, even much longer than the transcription time.

`model = WhisperModel(model_size, device="cuda", compute_type="float16")`

**Comment by michaelcrusso** [comment]
I think you need to provide some more information, or some output. 

- How long did the loading actually take? 
- What model are you attempting to load? 
- What GPU are you using? How much VRAM does it have?
- What CPU model do you have?
